test/integration/full_stack_test.rb

```diff
@@ -22,8 +21,7 @@ def setup
     Coverband.configuration.store.clear!
     Coverband.start
     Coverband::Collectors::Coverage.instance.runtime!
-    @rack_file = File.expand_path(TEST_RACK_APP, File.dirname(__FILE__))
-    require @rack_file
+    @rack_file = require_unique_file 'fake_app/basic_rack.rb'
```

Comment:
@danmayer - was running into sporadic errors. Making use of `require_unique_file` seems to clear things up.